ts/boot/configureStoreAndPersistor.ts

```diff
@@ -102,6 +102,21 @@ const migrations: MigrationManifest = {
       : {
           ...state
         };
+  },
+
+  // Version 5
+  // we empty the services list to get both services list and services metadata being reloaded and persisted
+  "5": (state: PersistedState) => {
+    return {
+      ...state,
+      entities: {
+        ...(state as any).entities,
+        services: {
+          ...(state as any).entities.services,
+          byId: {}
```

Comment:
This migration could be dangerous!
Indeed after migration your messages look like these "**_Mittente Sconosciuto_**"
<img src="https://user-images.githubusercontent.com/822471/63778162-9cc89300-c8e4-11e9-81d1-bc1f3f1c6ee6.png" height="600"/>

The messages senders will be loaded only after services screen will be visited and services will be loaded.
Anyway, since you are request only for visible services, if you have a message from a sender that is not longer visible you never have it back again, check it out the first message on this screen

<img src="https://user-images.githubusercontent.com/822471/63778383-f761ef00-c8e4-11e9-9e46-c14d740d2fa6.png" height="600"/>

Reply:
https://github.com/teamdigitale/io-app/commit/4a5010d53af5dcbf687bc50cc6ec1cff542f3554a adds control to refresh each service related to the available messages. 

Anyway, if the user goes from the messages home screen to the service home screen, all the services are refreshed. We could implement a logic that avoids the recently refreshed services to be refreshed again.